Dockerfile

```diff
@@ -1,24 +1,20 @@
 FROM golang:alpine as builder
 
 RUN apk --update --no-cache add make git g++ linux-headers
-# DEBUG
-RUN apk add busybox-extras
 
 ENV GO111MODULE on
 
-# Build statically linked vDB binary (wonky path because of Dep)
 WORKDIR /go/src/github.com/makerdao/vdb-mcd-transformers
 ADD . .
 
 WORKDIR /go/src/github.com/makerdao
 RUN git clone https://github.com/makerdao/vulcanizedb.git
 WORKDIR /go/src/github.com/makerdao/vulcanizedb
-RUN git checkout v0.0.10
```

Comment:
I'm trying to think if we'd need any sort of change to our workflow with pointing at VDB staging, in case we make a change to VDB that's incompatible with mcd transformers. e.g. the situation we'd be in if https://github.com/makerdao/vulcanizedb/pull/8 gets merged in before https://github.com/makerdao/vdb-mcd-transformers/pull/52

Reply:
yeah good point. I think this should probably be pointing at a release once a new one is cut. Maybe it would make sense to add an arg that you pass at build time to specify the branch/release of vdb you want to target?

Reply:
👍  I like that idea.